test/exercises_test.dart

```diff
@@ -28,7 +28,9 @@ Future runCmd(List<String> cmds) async {
 Future<String> getPackageName() async {
   final pubspec = new File("pubspec.yaml");
 
-  final packageName = loadYaml(await pubspec.readAsString())["name"];
+  final String sourceUrl = await pubspec.readAsString();
```

Comment:
This is what I was referring to my other comment. `sourceUrl` does not hold a `URL` but `pubspec` as a `String`. `pubspecString` would be a better name.

Reply:
Got it